eras/shelley/test-suite/cardano-ledger-shelley-test.cabal

```
@@ -87,13 +87,12 @@ library
         cardano-ledger-core:{cardano-ledger-core, testlib} >=1.0,
         cardano-ledger-pretty,
         cardano-ledger-shelley:{cardano-ledger-shelley, testlib} ^>=1.1,
-        cardano-protocol-tpraos >=1.0,
+        cardano-protocol-tpraos:{cardano-protocol-tpraos, testlib} >=1.1,
```

Here a lower bound indeed has to be updated, bit not to `1.1` (see other comment)
```suggestion
        cardano-protocol-tpraos:{cardano-protocol-tpraos, testlib} >=1.0.1,
```